src/App.jsx

```diff
@@ -2,17 +2,20 @@ import reactLogo from "./assets/react.svg";
 import viteLogo from "/vite.svg";
 import "./App.css";
 import { useState } from "react"; 
-import TodoList from "./TodoList";
+import TodoList from "./TodoList.jsx";
```

Comment:
When we use a bundler like Vite, we can typically forego the local extensions in our import statements: I.e.

```
import TodoList from './TodoList';
```

Since we're using Vite in this project, we can remove the .jsx file ending from this import.

Reply:
Thanks for your feedback!